sormas-backend/src/main/java/de/symeda/sormas/backend/person/PersonFacadeEjb.java

```diff
@@ -729,6 +818,11 @@ public static PersonDto toDto(Person source) {
 		return target;
 	}
 
+	// needed for tests
+	public void setExternalJournalService(ExternalJournalService externalJournalService) {
```

Comment:
The production code should not contain test code. 

You can let it if there is no other quick solution. Maybe would be nice to add a git issue for investigating mocking possibilities.

Reply:
This change was cherry-picked from another branch. I would prefer to leave it like that for now, although I think you have a valid point.